### DISCUSSION TITLE
Add to Benchmarks

### DISCUSSION THREAD

**Discussion by sirinath** [rwf2/Rocket#1680] [General]
Can this be added o benchmarks so it can be compared with the other frameworks:
- https://www.techempower.com/benchmarks/
- https://web-frameworks-benchmark.netlify.app/result?asc=0&order_by=level64
- etc.

**Comment by SergioBenitez** [comment]
I have no reservations about Rocket appearing in any benchmark suite, but it's not something I think the project should take on itself. See my previous comments about the matter in https://github.com/SergioBenitez/Rocket/issues/710#issuecomment-405721936.
